docs/tutorials/detached-mode/_index.rst

```diff
@@ -0,0 +1,40 @@
+.. _detached-mode-index:
+
+##########################
+ How to Use Detached Mode
+##########################
+
+Detached mode is an enhancement to the Core API that allows you to:
+
+-  Log metrics in Determined while running your workload *anywhere*.
+-  Maintain full control of your workload-associated metrics and metadata.
+-  Report metrics from multiple jobs rather than a single experiment.
+-  Track experiments using Determined notebooks.
```

Comment:
To match the other lists in this PR:
```suggestion
-  Log metrics in Determined while running your workload *anywhere*
-  Maintain full control of your workload-associated metrics and metadata
-  Report metrics from multiple jobs rather than a single experiment
-  Track experiments using Determined notebooks
```
I also think I prefer starting the lines with lowercase letters in this context, though I don't have a firm opinion or a certain understanding of the current prevailing style.

Reply:
the style i try to follow: if it is a sentence, add punctuation

Reply:
The thing is, these are clauses that happen to be viewable as full sentences, but in context, they're just parts of the bigger sentence, so I don't think that's the distinction to make. If this section were written as a paragraph rather than a bullet list, the items would be separated by commas just like the other lists, not sentences in their own right.

(I think this is another way of looking at it, if you want to get really into the weeds: these lines as individual sentences would be seen as containing the imperatives of the verbs, but with the "to" introducing the list, they're in fact the infinitive instead. Those happen to be spelled the same in English, but they're not quite the same.)

Reply:
@dzhu you are right, these are part of a sentence.